11-eos-storage.md

```diff
@@ -53,6 +53,9 @@ element on the Grid. For files which you do not plan to work with
 interactively and instead feed into a different grid job it makes
 sense to leave them on the grid.
 
+Note that EOS *IS* a grid storage (the CERN one), but your personnal space is just
+in a different subdirectory than the files "on the grid" managed by DIRAC.
+
```

Comment:
This is a nice addition, it should clarify some things. I would re-word slightly:

> Note that EOS _is_ 'Grid storage', specifically that which is located at CERN. Your personal space, at `/eos/lhcb/user/<your username's first letter>/<your username>`, is just another location on EOS, and happens to be different to the default location used by DIRAC for 'grid storage'.